src/base/zonefile_fmt.rs

```diff
@@ -11,21 +11,32 @@ impl From<fmt::Error> for Error {
 
 pub type Result = core::result::Result<(), Error>;
 
+pub enum DisplayKind {
+    Simple,
```

Comment:
I think "Simple" has the same problem as the term "pretty" in that it is not clear what it means and what one person considers simple another might not.

Reply:
Maybe in RustDoc on this type, or at the module level, we should show examples of the different formats side-by-side to make it easier for a user to compare them?